doc/Language/variables.pod6

```diff
@@ -528,8 +527,8 @@ variables.
     say [$a, $b, $c].perl;
     # OUTPUT«["a", "b", Int]␤»
 
-To destructure a list into a single value create a list literal with one value
-by using C<($var,)>. When used with a variable declarator just providing
+To destructure a list into a single value, create a list literal with one value
```

Comment:
So:
To destructure a list into a single value, create a list literal with one element.